### PR TITLE
Add ansible docs

### PR DESCRIPTION
**Pull request by SusiJo**
This documents how to install different pieces of software on a server with ansible. It greatly simplifies the installation of software packages like docker, singularity, java, bash cmd tools, etc.

### CODE REVIEW

File: docs/markdown/clusters/denbi_cloud.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ If you haven't created an instance based on an Image that already has java, Next
   source ~/.bashrc
   java --version
   ```
-
+
```
Comment (SusiJo) [inline docs/markdown/clusters/denbi_cloud.md]:
```suggestion

```

File: docs/markdown/clusters/denbi_cloud.md [hunk 2]
```diff
@@ -147,3 +147,87 @@ screen -S newscreen
...
+      lineinfile:
+        dest: /home/centos/.bashrc
+        line: export JAVA_HOME=/opt/jdk-17 && export PATH=$PATH:$JAVA_HOME/bin
+```
+
+- Copy the lines below into a file `installation_instructions.sh` and execute on server with `sh installation_instructions.sh`
+
+```bash
+# ** IMPORTANT NOTE **
+# Make sure to select right device : /dev/vdb or /dev/vdc ...
```
Comment (louperelo) [inline docs/markdown/clusters/denbi_cloud.md:192]: How can you check, which one is right? Does only one of them exist?

Comment (louperelo) [reply docs/markdown/clusters/denbi_cloud.md:192]: Depending on this selection you will have to adapt the command `sudo mkfs.ext4 /dev/vdb` in line 199 and 205, right?

**Review by louperelo — APPROVED** [review]
Looks good to me.

File: docs/markdown/clusters/denbi_cloud.md (hunk 2 above)
Comment (SusiJo) [reply docs/markdown/clusters/denbi_cloud.md:192]: Yes that's correct. You can check which devices are available with `lsblk`. If there's only one volume attached, the default is `/dev/vdb` and if one attaches a second volume it is `/dev/vdc`.

**Pull request merged**